common/qconnector/__init__.py

```
@@ -0,0 +1,2 @@
+from .q_connector import QConnector
```

Ruff (0.11.9)

`.q_connector.QConnector` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)